pyminion/core.py

```diff
@@ -289,21 +294,44 @@ def pile_length(self, pile_name: str) -> int:
 
 
 def get_action_cards(cards: Iterable[Card]) -> Iterator[Action]:
+    """
+    Gets an iterator over the action cards in a Card iterable.
```

Comment:
Super minor and nitty but can you change "Gets" to "Returns" in these comments? I know the function name contains "get", but I think "return" is the more natural way to comment the behavior.

Reply:
Fixed!